document/CoreFormat.mdk

```diff
@@ -1107,8 +1107,10 @@ A `fileChange` object represents a change to a single file.
 
 ### `uri` property { #fileChange-uri }
 
-A `fileChange` object shall contain a property named `uri` whose value is a string value that represents
-the location of the file as a valid URI.
+A `fileChange` object may contain a property named `uri` whose value is a string value that represents
+the location of the file as a valid URI. If the 'uri' property is not present, its value is assumed
+to be the uri that is associated with the current result (i.e., the uri present on the result 
```

Comment:
I am fine leaving this as it is for now (requiring the value) based on your feedback. The format clearly isn't designed to be compact in any case, so tuning for this scenario doesn't seem high priority. It'd be good to solicit some feedback from the reviewers who proposed the fix extension to the spec.